doc/source/reference/random/index.rst

```diff
@@ -0,0 +1,219 @@
+.. currentmodule:: numpy.random
+
+numpy.random
+============
+
+A `~RandomGenerator` can
+be initialized with a number of different Random Number Generators (RNG)s, and
+exposes many different probability distributions.
+
+
+Quick Start
+-----------
+
+By default, `RandomGenerator` uses normals provided by
+`xoroshiro128.Xoroshiro128` which will be faster than the legacy methods in
+`mtrand.RandomState`
+
+.. code-block:: python
+
+  # As replacement for numpy.random.RandomState
+  from numpy import random
+  random.standard_normal()
+
+`RandomGenerator` can be used as a direct replacement for
+`~RandomState`, although the random values are generated by
+`~xoroshiro128.Xoroshiro128`. The `RandomGenerator` holds an instance of a RNG.
+It is accessable as ``gen.brng``.
+
+.. code-block:: python
+
+  # As replacement for RandomState()
+  from numpy.random import RandomGenerator
+  rg = RandomGenerator()
+  rg.standard_normal()
+
+
+Seeds can be passed to any of the basic RNGs. Here `mt19937.MT19937` is used
+and the ``RandomGenerator`` is accessed via the attribute `mt19937.MT19937.
+generator`.
+
+.. code-block:: python
+
+  from numpy.random import MT19937
+  rg = MT19937(12345).generator
+  rg.standard_normal()
+
+
+Introduction
+------------
+RandomGen takes a different approach to producing random numbers from the
+:class:`numpy.random.RandomState` object.  Random number generation is
+separated into two components, a basic RNG and a random generator.
```

Comment:
This introduction is written a little strangely for NumPy documentation. It assumes the user is already familiar with the legacy numpy.random module.

Reply:
This is vestigial.